modules/package.nix

```diff
@@ -132,6 +132,11 @@ in {
         default = false;
       };
 
+      isProjectPackage = mkOption {
```

Comment:
Maybe `isProject` for symmetry with `isLocal`?

Reply:
The difference is that `isLocal` includes `source-repository-packages`.

Reply:
I just meant the name of the option. We have `isLocal` and `isProjectPackage`. Arguably we should rename `isLocal` to be more descriptive, I don't mind but I think they should be symmetrical!

Reply:
I've gone with `isProject` for now (renaming `isLocal` would require updating `nix-tools` and rematerializing everything).